src/definitions.ts

```diff
@@ -381,6 +383,8 @@ export interface CapacitorCalendarPlugin {
 
   /**
    * Deletes events from the calendar given their IDs.
+   * If the event is recurring it will automatically delete this and future events.
```

Comment:
This might lead to unwanted behaviors. Could you add an optional `span` property for being able to choose between this or all future events to be deleted?

Reply:
The problem kind of is that this only needs to be supplied for recurring events.
It would be possible to add an `EventSpan` array but this would have a lot of overhead for the caller of the function.

Maybe we could think of a different approach?